utils/swift_build_support/swift_build_support/products/benchmarks.py

```diff
@@ -102,9 +102,7 @@ def _get_toolchain_path(host_target, product, args):
     # this logic initially was inside run_build_script_helper
     # and was factored out so it can be used in testing as well
 
-    toolchain_path = swiftpm.SwiftPM.get_install_destdir(args,
-                                                         host_target,
-                                                         product.build_dir)
+    toolchain_path = product.host_install_destdir(host_target)
```

Comment:
This isn't actually going to work for cross-compiling for non-Darwin so I looked into using `install_toolchain_path`, like I did with indexstoredb below, but the concatenation below appears to be slightly different than what that method does. Since I don't want to break the Darwin config and cross-compiling these benchmarks for non-Darwin would require other changes anyway, this should be fine for now.

Reply:
I am fine with this if the tests pass (which it looks like they did).